Gcpe.Hub.API.IntegrationTests/SecuredPagesShould.cs

```diff
@@ -1,17 +1,33 @@
-﻿using System.Net;
+﻿using System.IO;
+using System.Net;
 using System.Net.Http;
 using System.Threading.Tasks;
 using FluentAssertions;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.Mvc.Testing;
 using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.Configuration;
 using Xunit;
 
 namespace Gcpe.Hub.API.IntegrationTests
 {
     public class SecuredPagesShould : BaseWebApiTest
     {
-        public SecuredPagesShould(CustomWebApplicationFactory<TestStartup> factory) : base(factory) { }
+        private HttpClient client;
+        public SecuredPagesShould(CustomWebApplicationFactory<TestStartup> factory) : base(factory)
+        {
+            string contentRoot = Directory.GetParent(Directory.GetCurrentDirectory()).Parent.Parent.Parent.GetDirectories("Gcpe.Hub.Api")[0].FullName;

```

Comment:
Wondering if there's a more concise way of doing this? If not, that's cool.

Reply:
I got this bit from Bruno and it's done like this in the AzureSearchService project. Alternatively we could add appsettings.json to the Integration tests folder but then we are repeating ourselves